test/next/src/app/vercel/blob/script.mts

```diff
@@ -39,10 +39,18 @@ async function run(): Promise<void> {
     manualMultipartUploader(),
   ]);
 
+  // multipart uploads are frequently not immediately available so we have to wait a bit
+  await new Promise((resolve) => setTimeout(resolve, 5000));
+
   await Promise.all(
     urls.map(async (url) => {
-      const blobDetails = await vercelBlob.head(url);
-      console.log(blobDetails, url);
+      try {
+        const blobDetails = await vercelBlob.head(url);
+        console.log(blobDetails, url);
+      } catch (err) {
+        console.log('url', err);
+        process.exit(1);
```

Comment:
why do we exit here? I think the script should still try to delete the files.

Reply:
I can remove that, used only for testing